app/services/analytics_events.rb

```diff
@@ -3547,42 +3547,48 @@ def idv_in_person_usps_request_enroll_exception(
     )
   end
 
-  # GetUspsProofingResultsJob is beginning. Includes some metadata about what the job will do
+  # USPS proofing results job is beginning.
+  # Includes some metadata about what the job will do.
   # @param [Integer] enrollments_count number of enrollments eligible for status check
   # @param [Integer] reprocess_delay_minutes minimum delay since last status check
+  # @param [String] name of the job that triggered this event
   def idv_in_person_usps_proofing_results_job_started(
-    enrollments_count:,
-    reprocess_delay_minutes:,
+    enrollments_count: nil,
+    reprocess_delay_minutes: nil,
+    job_name: nil,
```

Comment:
i would think we never want the name to be nil

Reply:
Yeah, it shouldn't ever be. In theory it could be if it wasn't passed in. I'm trying to match the existing convention but maybe I'm misunderstanding it?